db/migrate/20230503123549_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :author_id
+      t.integer :post_id
+      t.date :createdat
+      t.date :updatedat
```

Comment:
- [optional] The createdat and updatedat columns are automatically created by rails  using the `t.timestamps`. Kindly remove these please.